src/ci/github-actions/ci.yml

```diff
@@ -264,6 +264,9 @@ on:
     branches:
       - "**"
 
+permissions:
+  contents: read
```

Comment:
There is an action that uses the `actions: write` permission in our workflow:

https://github.com/rust-lang/rust/blob/5fecdb7c0ab4fb26f37df7c5e6afb4981e0fc71b/src/ci/github-actions/ci.yml#L133-L138

That whole action can probably be replaced with the [`concurrency`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#concurrency) key though (we created the action before `concurrency` was introduced).

Reply:
Thank you, I missed that. To keep one issue per pull request, I only added the permission for now.